guides/plugins/plugins/administration/overriding-routes.md

```diff
@@ -0,0 +1,59 @@
+# Overriding routes
+
+## Overview
+
+In the `Administration` core code, each module is defined in a directory called `module`. Modules define routes wich can be extended with `routeMiddleware`.
+Learn what else you can customize, in existing modules in this [guide](./customizing-modules.md)
+
+A `module` is an encapsulated unit which implements a whole feature. For example there are modules for customers, orders, settings, etc.
+
+## Prerequisites
```

Comment:
Aren't we having a basic modules guide as a prerequisite here ?

Reply:
probably a good idea